### PR TITLE
Remove redundant `self.built=true` from keras/layers/convolutional.py

### PR DESCRIPTION
**Pull request by yd8534976**
In `_maybe_build()`, `self.built` will be set `true` automatically. So all `self.built=true` of `Conv` will be redundant. This PR removes them. https://github.com/tensorflow/tensorflow/blob/75a91fac16493b6f68c0740b214ff18f5d88e5a8/tensorflow/python/keras/engine/base_layer.py#L2144

### CODE REVIEW

**Comment by yd8534976** [comment]
@googlebot I signed it!

**Comment by gbaned** [comment]
@yd8534976 Can you please resolve conflicts? Thanks!

**Comment by yd8534976** [comment]
@gbaned Done. Thanks!

**Comment by gbaned** [comment]
@tanzhenyu Can you please take a look on this PR? Thanks!

**Review by mrinal18 — APPROVED** [review]

**Review by pavithrasv — APPROVED** [review]

**Comment by gbaned** [comment]
@yd8534976 Can you please check build failures? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
